Generate/src/Medication-9-0-7/PHR-Client/AdministrationAgreement/medmij-medication-administrationagreement-phr-1-2.xml

```diff
@@ -6,70 +6,25 @@
     <description value="Scenario 1.2 - Specifieke toedieningsafspraken met hun identificaties als filter."/>
     <nts:patientTokenFixture href="mp9-nl-core-patient-XXX-DIRKSZ-token.xml" />
     
-    <test id="Scenario-1-2-1">
-        <name value="Scenario 1.2 test 1"/>
+    <test id="Scenario-1-2">
+        <name value="Scenario 1.2"/>
         <description value="Specifieke toedieningsafspraken met hun identificaties als filter."/>
         <nts:include value="operation-search">
             <nts:with-parameter name="description" value="Query MedicationDispense resource(s) for AdministrationAgreement"/>
             <nts:with-parameter name="resource" value="MedicationDispense"/>
-            <nts:with-parameter name="params" value="?category=http://snomed.info/sct|422037009&amp;identifier=urn:oid:2.16.840.1.113883.2.4.3.11.999.77.422037009.1|MBH_113_1_TA_1&amp;_include=MedicationDispense:medication"/>
+            <nts:with-parameter name="params" value="?category=http://snomed.info/sct|422037009&amp;identifier=urn:oid:2.16.840.1.113883.2.4.3.11.999.77.422037009.1|MBH_113_1_TA_1,urn:oid:2.16.840.1.113883.2.4.3.11.999.77.422037009.1|MBH_113_2_TA_1,urn:oid:2.16.840.1.113883.2.4.3.11.999.77.422037009.1|MBH_113_3_TA_1&amp;_include=MedicationDispense:medication"/>
```

Comment:
Ik kan comma-separated search params alleen vinden in de R4 search spec (https://www.hl7.org/fhir/search.html#combining), niet in de STU3. Ik neem aan dat het getest is?

Reply:
Dit passen we ook toe in de BgZ queries. Zie afbeelding. 

![image](https://user-images.githubusercontent.com/20518398/104742941-61359f80-574b-11eb-8ab6-dd0a553a82dd.png)

Verder staat dti ook in STU3, https://www.hl7.org/fhir/stu3/search.html#combining:
"If, instead, the search is to find patients that speak either language, then this is a single parameter with multiple values, separated by a ,. For example, /Patient?language=FR,NL. This is known as an OR search parameter, since the server is expected to respond with results which match either value."

Reply:
En heb het getest 👍